core/tests/test_utils.py

```diff
@@ -143,12 +158,26 @@ class TestBase(unittest.TestCase):
     }
 
     def _get_unicode_test_string(self, suffix):
+        """ Supply a test string that has unicode characters in it.
+
+        Args:
+            suffix: str. The suffix to UNICODE_TEST_STRING.
```

Comment:
I don't think I quite understand this description. When you look at the callers, what is the point of the suffix? Is it tied to UNICODE_TEST_STRING or is it just "A suffix to append to the UNICODE_TEST_STRING."?